python/letsql/backends/let/tests/test_cache.py

```diff
@@ -553,3 +555,32 @@ def test_replace_table_matching_kwargs(pg, ls_con, tmp_path):
     )
 
     assert expr.ls.native_expr is not None
+
+
+def test_cache_default_path_set(pg, ls_con, tmp_path):
+    letsql.options.cache_default_path = tmp_path
+
+    storage = ParquetCacheStorage(
+        source=ls_con,
+        path=tmp_path,
```

Comment:
`path` should not be passed here so we can verify that modification of `letsql.options.cache_default_path` takes effect

Reply:
Done